geoalchemy2/types/__init__.py

```diff
@@ -114,14 +115,14 @@ class _GISType(UserDefinedType):
 
     def __init__(
         self,
-        geometry_type="GEOMETRY",
+        geometry_type: Optional[str] = "GEOMETRY",
         srid=-1,
         dimension=2,
         spatial_index=True,
         use_N_D_index=False,
-        use_typmod=None,
-        from_text=None,
-        name=None,
+        use_typmod: Optional[Any] | Literal[False] = None,
```

Comment:
Why not just `Optional[bool]` here?

Reply:
Technically, it's more accurate, but I did not realize it is not available in `3.7`, bool it is!